compat/src/index.d.ts

```diff
@@ -106,7 +106,7 @@ declare namespace React {
 		component: preact.Component | Element
 	): Element | null;
 
-	export abstract class PureComponent<P = {}, S = {}> extends preact.Component<
+	export abstract class PureComponent<P = {}, S = {}, SS = any> extends preact.Component<
```

Comment:
Should we then also accept this on Component to orchestrate typings for `getSnapshotBeforeUpdate` as React does

Reply:
Sorry, I actually have no idea about any of that as I'm pretty new to using React/Preact, just thought I'd contribute a fix to get Blueprint working since it's just a small issue with types.